### ISSUE TITLE
time to migrate bug-tracking to the new projects...objections?

### ISSUE THREAD

**Issue by cmadland** [TWUOnline/bug-tracking#656]

**Comment by kmarjanovic** [comment]
Makes sense to me that issues are attached to projects...at the same time, will we lose the big picture of the project board that helps to prioritize?

Whatever will help Production - I'm all for it!! :)
